#!/usr/bin/env python
PACKAGE = "vdb_mapping_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_range", double_t, 0, "max_range parameter", 10.0, 0.0, 100.0)
gen.add("lower_visualization_z_limit", double_t, 0, "Lower z visualization bound", 0.0, -100.0, 100.0)
gen.add("upper_visualization_z_limit", double_t, 0, "Upper z visualization bound", 0.0, -100.0, 100.0)
gen.add("publish_pointcloud", bool_t, 0, "publish_pointcloud parameter",  True)
gen.add("publish_vis_marker", bool_t, 0, "publish_vis_marker parameter",  True)
gen.add("publish_updates", bool_t, 0, "publish_updates parameter",  False)
gen.add("publish_overwrites", bool_t, 0, "publish_overwrites parameter",  False)

exit(gen.generate(PACKAGE, "vdb_mapping_ros", "VDBMappingROS"))
